src/Glpi/Helpdesk/DefaultDataManager.php

```diff
@@ -165,6 +165,17 @@ private function addFormTranslations(int $forms_id, string $name, string $descri
         $description_hash = md5($description);
         $section_name_hash = md5($first_section->fields['name']);
         foreach (array_keys($CFG_GLPI['languages']) as $lang) {
+            $DB->insert(
+                ItemTranslation::getTable(),
+                [
+                    'itemtype'     => Form::class,
+                    'items_id'     => $forms_id,
+                    'language'     => $lang,
+                    'key'          => Form::TRANSLATION_KEY_NAME,
+                    'translations' => '{}',
+                ]
+            );
```

Comment:
Maybe I miss something, but as far as there is a unique key `unique_item_key` (`items_id`, `itemtype`, `key`, `language`), adding this entry should make the `$form_stmt->execute([Form::TRANSLATION_KEY_NAME, $lang, json_encode(['one' => $translated_name]), $name_hash]);` query fail.